src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -92,13 +92,16 @@ static int nbd = -1;
 #endif
 #define htonll(a) ntohll(a)
 
-static int do_unmap();
-
```

Comment:
You added this in the previous commit and then removed here.

I still think that just squashing two commits into one would be better. Anyway, if you want two commits, please clean up this.